.env.example

```diff
@@ -45,23 +44,7 @@ GATSBY_VOTING_POWER_TO_PASS_DRAFT=0
 GATSBY_VOTING_POWER_TO_PASS_GOVERNANCE=0
 
 # Grant sizes for each tier
-GATSBY_GRANT_SIZE_MINIMUM=100
-GATSBY_GRANT_SIZE_TIER1=1500
-GATSBY_GRANT_SIZE_TIER2=3000
-GATSBY_GRANT_SIZE_TIER3=5000
-GATSBY_GRANT_SIZE_TIER4=60000
-GATSBY_GRANT_SIZE_TIER5=120000
-GATSBY_GRANT_SIZE_TIER6=240000
-
-# Override VP for Grant Tiers, defaults to GATSBY_VOTING_POWER_TO_PASS_GRANT
-GATSBY_VOTING_POWER_TO_PASS_GRANT_TIER4=600
-GATSBY_VOTING_POWER_TO_PASS_GRANT_TIER5=700
-GATSBY_VOTING_POWER_TO_PASS_GRANT_TIER6=800
-
-# Override Duration for Grant Tiers, defaults to GATSBY_SNAPSHOT_DURATION
-GATSBY_DURATION_GRANT_TIER4=1200
-GATSBY_DURATION_GRANT_TIER5=1200
-GATSBY_DURATION_GRANT_TIER6=1200
+GATSBY_GRANT_SIZE_MINIMUM=1
```

Comment:
We changed the minimum to 100, right?

Reply:
We are not using this constant anymore, we replaced it by:

![image](https://user-images.githubusercontent.com/2858950/220187476-3b3668d5-85eb-4aed-ae87-8f55af37d337.png)

We could add them to the configs json but it's not like we need different configurations per environment. So we could just remove it and leave those there, WDYT?

Reply:
https://github.com/decentraland/governance/pull/740